Mathlib/GroupTheory/GroupAction/Defs.lean

```diff
@@ -666,6 +679,13 @@ theorem smul_one_smul {M} (N) [Monoid N] [SMul M N] [MulAction N α] [SMul M α]
 #align smul_one_smul smul_one_smul
 #align vadd_zero_vadd vadd_zero_vadd
 
+@[to_additive]
+theorem MulAction.IsPretransitive.of_isScalarTower
+    {M N α : Type*} [Monoid M] [Monoid N] [MulAction M α] [MulAction N α] [MulAction M N]
+    [IsScalarTower M N α] [IsPretransitive M α] :
+    IsPretransitive N α :=
```

Comment:
You can use the same TC assumptions as in `smul_one_smul`. Otherwise LGTM. Thanks!
bors d+
```suggestion
theorem MulAction.IsPretransitive.of_isScalarTower (M : Type*) {N α : Type*} [Monoid N] [SMul M N]
    [MulAction N α] [SMul M α] [IsScalarTower M N α] [IsPretransitive M α] :
    IsPretransitive N α :=
```